CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0]
+
+### Added
+
+- Add `module_enabled`, `module_depends_on`, `module_tags` and all input variables as outputs
+
+### Changed
+
+- **BREAKING**: Move `ecr:ListImages` from `push_identities` instead of `push_identities`.
+  Identities passed to `push_identities` will no longer be able to list images of a repository.
+
+### Removed
+
+- Remove redundant `ecr:GetAuthorizationToken` from `aws_ecr_repository_policy`
```

Comment:
this is not a breaking change since the policy didn't take effect in the first place (it needs to be applied to the * resource in order to work)